README.md

```diff
@@ -1,8 +1,43 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git
+
+**Competent:** Docker, Maven, REST API, JDBC, Spring Boot
+
+**Familiar:** Matlab, Python, ReactJS, MongoDB, Node.js
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi](https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi/tree/master/linux_sql)]: The Objective of this project was to design an MVP in order to gather hardware specifications and resource usage of each node in a Linux cluster. In Jarvis, the Linux cluster administration team manages a Linux cluster of ten nodes running Centos7. These nodes are internally connected through a switch and able to communicate through internal IPv4 addresses. The administration team has requested a bash agent program that is installed on each node and gathers the requested data. The collected data is inserted into a Postgres database instance which is pulled from Docker Hub. Moreover, the LCA team will use the data to generate reports for future resource planning purposes. In order to make decision making easier, a queries file has been created for the administration team to see what nodes have the highest memory and number of CPUs, what is their usage over a five minute interval and detecting failures.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_AliSaboohi/tree/master/core_java)]:
+      
+  - Twitter App: 
+  - JDBC App: 
+  - Grep App: 
+
+
+## Highlighted Projects
+**Clothing Store Web Application**: Designed and Developed Online clothing Web site with Online Shopping facility. using Java and Spring Boot for backend and React Js for Frontend. Frontend developed as SPA application in order to use Client Computer Resources using JavaScript ES6 Version, Vanilla JavaScript, Babel, React-Dom-route, Redux, HTML and CSS, in Backend using Spring Core for dependency Injection and AOP, Spring MVC to develop Rest Endpoints (RestController Bean) and JSON. implementing Data Entity Classes and using  Hibernate, Spring Data JPA, JPA QL for managing Data and store it in MySQL Database
```

Comment:
Designed and Developed an online clothing e-commerce website. The frontend was developed using JavaScript ES6 Version, Vanilla JavaScript, Babel, React-Dom-route, Redux, HTML and CSS. The backend was developed using Spring Core for dependency Injection and AOP, Spring MVC to build Rest Endpoints and JSON. Implemented Data Entity Classes using Hibernate, Spring Data JPA, JPA QL to manage and store data in MySQL Database.